frontend/integration-tests/tests/overview/overview.scenario.ts

```diff
@@ -37,6 +39,22 @@ describe('Visiting Overview page', () => {
         expect(overviewView.getProjectOverviewListItem(kindModel, resourceName).isPresent()).toBeTruthy();
       });
 
+      it('clicking on N of N pods takes you to Pods page', async() => {
+        await overviewView.ClickOverviewItemsDetailsPodsLink(kindModel);
+        if ( kindModel.kind === 'DaemonSet' || kindModel.kind === 'StatefulSet') {
```

Comment:
I'm not sure we want to have so much special logic around checking the URLs. It might be simpler to check if we are on a pods tab pages, and this would work generically for all types.

Reply:
checking on a pods tab page is much simpler, ok let me update